### PR TITLE
Object bounds

### PR DESCRIPTION
**Pull request by susuhahnml**

### CODE REVIEW

File: ooasp_configuration.lp
Comment (rtaupe) [inline ooasp_configuration.lp:1]: What is this file why has it changed?
Should it really be under version control?

File: ooasp_configuration_wrong.lp
Comment (rtaupe) [inline ooasp_configuration_wrong.lp:1]: What is this file why has it changed?
Should it really be under version control?

File: ooasp/encodings/ooasp_guess.lp [hunk 1]
```diff
@@ -1,6 +1,17 @@
 % Copyright (c) 2022 Siemens AG Oesterreich
 % SPDX-License-Identifier: MIT
 
+#program include(new_object, cls).
+
+% Include for performance
+ooasp_isa(cls,new_object).
+
+% Fact used in the UI to indentify objects that must be included
```
Comment (rtaupe) [inline ooasp/encodings/ooasp_guess.lp]:
```suggestion
% Fact used in the UI to identify objects that must be included
```

**Review by rtaupe — CHANGES_REQUESTED** [review]
Thank you!
I just have a few questions to address, please.

File: ooasp_configuration.lp
Comment (susuhahnml) [reply ooasp_configuration.lp:1]: This two files were generated output. I just removed them

**Review by rtaupe — APPROVED** [review]
Thank you!

**Pull request merged**